file-formats/srvc/examples/z_aff_example_srvc.srvc.json

```
@@ -0,0 +1,13 @@
+{
+    "formatVersion": "1",
+    "header": {
+      "description": "Example SRVC for ABAP File Formats",
+      "originalLanguage": "en"
+    },
+    "consumerInformation": {
+      "consumerType": "odata",
+      "consumerObjectName": "SRVC_ODATA",
+      "consumerObjectType": "SRVD",
+      "connectionProperties": []
+    }
```

Just to get you an overview about my ideas:

```suggestion
    "consumer": {
      "type": "OData",
      "objectName": "SRVC_ODATA",
      "objectType": "SRVD"
    }
```

Maybe you want even to change the order of fields `objectType` and `objectName`?

---

Changed the order of fields.